packages/service-core/src/routes/endpoints/sync-rules.ts

```diff
@@ -1,12 +1,12 @@
-import * as t from 'ts-codec';
-import type { FastifyPluginAsync } from 'fastify';
-import * as pgwire from '@powersync/service-jpgwire';
 import { errors, router, schema } from '@powersync/lib-services-framework';
 import { SqlSyncRules, SyncRulesErrors } from '@powersync/service-sync-rules';
+import type { FastifyPluginAsync } from 'fastify';
+import * as t from 'ts-codec';
 
-import * as replication from '../../replication/replication-index.js';
+import * as system from '../../system/system-index.js';
```

Comment:
Is this still used?

Reply:
Removed in https://github.com/powersync-ja/powersync-service/pull/137